src/bootstraps/core/graphistry.sh

```diff
@@ -29,7 +29,6 @@ echo
 ./prepopulate-notebooks.sh graph-app-kit/public views ubuntu
 ./prepopulate-notebooks.sh graph-app-kit/private views ubuntu
 ./graphistry-wait-healthy.sh
-./swap-caddy.sh
```

Comment:
I think almost all of this file can go away as Graphistry has gak pub/priv built in, in unclear what's left here?